wallet/res/layout/transaction_row.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I changed to ConstraintLayout because I couldn't figure out how to make the primary status text end in ... while keeping the secondary text from getting cropped or overlapped.